### DISCUSSION TITLE
Did you plan to be able to run in pipeline?

### DISCUSSION THREAD

**Discussion by dongokovacs** [usebruno/bruno#1034] [Ideas]
Hi @helloanoop your work is amazing, i use it almost every day.
Do you plan to be able Bruno to run in pipeline (aure devops...)?

**Comment by helloanoop** [comment]
Hey @daniavander !

Thanks for the kind words :)

Yes, you can run your collections in any CICD pipeline. See docs: https://docs.usebruno.com/cli/overview.html

**Comment by ashish-mountblue-07** [comment]
Hi @daniavander tried to check the doc you mentioned above the page isn't available can you please check.

**Reply by sanjai0py** [reply]
Here is the updated link: https://docs.usebruno.com/bru-cli/overview
